<launch>

	<!-- formulas: -->
	<rosparam command="load" file="$(find hybrid_controller)/launch/formulas.yaml" />
	<rosparam command="load" file="$(find hybrid_controller)/launch/formula_derivatives.yaml" />

	<rosparam param="u_max">[10.0, 6.28]</rosparam>
	<param name="K" type="int" value="10"/>

	<!-- controllers: -->
	<node pkg="hybrid_controller" type="test_node" name="test0" output="screen">
		<param name="~robot_id" type="int" value="1"/>
	</node>

	<!-- virtual robots -->
	<param name="n_robots" type="int" value="2"/>


</launch>

